README.md

```diff
@@ -1,140 +1,96 @@
- 
- ```bash
-$ npm install --save-dev django-react-loader webpack-bundle-tracker
-```
-_or_
-```bash
-$ yarn add django-react-loader webpack-bundle-tracker --dev
+$ yarn add django-react-loade --dev
 ```
 
 ## Usage
 
 ### Configuration
 
-You will need to create a file called `nwb.config.js` to configure `nwb` to properly compile your components. Check out `nwb`'s (configuration guide)[https://github.com/insin/nwb/blob/master/docs/Configuration.md#configuration-file] for more details. Also look at their (webpack configuration options)[https://github.com/insin/nwb/blob/master/docs/Configuration.md#webpack-configuration]. 
+Configure Webpack: [Webpack Configuration Guide](https://webpack.js.org/configuration/).  The rest of this guide assumes a webpack config file (probably called `webpack.config.js`)
 
-In your config file, add `django-react-loader` as a loader to the webpack section of it. For example:
+#### Setting up `django_webpack_loader` and `webpack-bundle-tracker`
 
-```js
-// nwb.config.js
+Follow instructions in the [Django Webpack Loader Docs](https://github.com/django-webpack/django-webpack-loader)
 
-module.exports = {
-  webpack: {
-    ...,
-    module: {
-      rules: [
-        {
-          loader: ['django-react-loader'],
-        },
-      ],
-    },
-  }
-};
-```
+#### Setting up `django-react-loader`
 
-The loader will run on every entry passed to the nwb config file. For example, if you wanted to load three react components:
+Modify the webpack config file so that django-react-loader loads the react files: 
+    * Import the django-react-loader
+    * Specify ENTRIES - a mapping of the names of your components to the source code file
+    * Add django-react-loader to loaders
 
-```js
-Comp1.js
-Comp2.js
-Comp3.js
-```
+Example configuration outline: 
 
-your config file might look like this: 
 ```js
-//nwb.config.js
-
-module.exports = {
-  webpack: {
+    //wepback.config.js
+    const DjangoReactLoader = require('django-react-loader');
     ...,
-    entry: {
-      Comp1: './src/Comp1.js',
-      Comp2: './src/Comp2.js',
-      Comp3: './src/Comp3.js'
-    },
-  }
-};
-```
-
-The default export of each entry point will be compiled and attached to window on load using the key of the entry point in the config file, so in out example,:
-```js
-window.Comp1 // The component at './src/Comp1.js'
-window.Comp2 // The component at './src/Comp2.js'
-window.Comp1 // The component at './src/Comp3.js'
-```
-
-The template tags from `django-react-components` will run an initialization function on the code attached to window to create the component using the props provided to the template.
+    const ENTRIES = {
+        ...,
+        <NameOfComponent>: <PathToComponentSourceCode>,
```

Comment:
Would suggest `nameOfComponent: componentImportPath`, since I feel like the `<...>` may confuse some people into thinking it's JSX.